File: block_plugs/biubiu_home/src/main/java/com/leichao/biubiu/home/HomeApplication.java
package com.leichao.biubiu.home;

import android.app.Application;

import com.leichao.util.AppUtil;

public class HomeApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        // 初始化Util工具
        AppUtil.init(this);
    }

}

File: block_plugs/biubiu_home/src/main/java/com/leichao/biubiu/home/HomeApplication.kt
package com.leichao.biubiu.home

import com.leichao.common.base.BaseApplication

import com.leichao.util.AppUtil
import com.qihoo360.replugin.RePlugin

class HomeApplication : BaseApplication() {

    override fun onCreate() {
        super.onCreate()
        AppUtil.init(this, RePlugin.getHostContext()) // 初始化Util工具
    }

}
